index.html

```diff
@@ -5,157 +5,190 @@
     </head>
 
 
     <body>
         <header class="main-header">
             <nav class="header">
-                <div class="name"><a href="#" style="color: white; text-decoration: none;">Usama Masood</a></div>
+                <div class="name"><a class="nam" href="#" style="color: white; text-decoration: none;">Usama Masood</a></div>
                 <div class="menu"><i class="fas fa-bars"></i></div>
+                <div class="men">
+                    <div class="port">Porfolio</div>
+                    <div class="ab">About</div>
+                    <div class="co">Contact</div>
+                    <div class="mail"><img src="./icons/Disabled.png" alt=""></div>
+                </div>
             </nav>
         </header>
 
 
         <main class="main1">
             <section class="section1">
-                <div class="gap">gap</div>
-                <div class="intro">
-                    <h1>Hey There. <br>
-                        I'm Usama
-                    </h1>
-                </div>
-                <div class="profession"><h2>I'am a Software Developer</h2></div>
-                <div class="containe">
-                    <div class="containe1">
-                        <div>
-                            <article class="services">
-                                <p>
-                                    I can help you build a product , feature or <br>
-                                    website Look through some of my work <br>
-                                    and experience! If you like what you see <br>
-                                    and have a project you need coded, <br>
-                                    don’t hestiate to contact me. <br>
-                                </p>
-                            </article>
-                            <ul class="logo">
-                                <li><a href="https://github.com/UsamaMasood12"><img src="./image/G.png" alt=""></a></li>
-                                <li><a href="www.linkedin.com/in/usama-masood-b4a35014b"><img src="./image/Group.png" alt=""></a></li>
-                                <li><a href="#"><img src="./image/M.png" alt=""></a></li>
-                                <li><a href="https://twitter.com/Usama____Masood"><img src="./image/twitter.png" alt=""></a></li>
-                                <li><a href="#"><img src="./image/V.png" alt=""></a></li>
-                            </ul>
+                <div class="aaa">
+                    <div class="aaaa">
+                        <div class="intro">
+                            <div class="gap"></div>
+                            <h1>Hey There. <br>
+                                I'm Usama
+                            </h1>
                         </div>
+                        <div class="profession"><h2>I'am a Software Developer</h2></div>
```

Comment:
- [ ] Great job 👍 You gave this text a `font-family` of `Roboto` instead of `Inter`. Kindly fix this 🙏